python/graphstorm/model/lp_gnn.py

```diff
@@ -78,6 +113,26 @@ def __init__(self, alpha_l2norm):
         super(GSgnnLinkPredictionModel, self).__init__()
         self.alpha_l2norm = alpha_l2norm
 
+    def prepare_pos_graph(self, pos_graph, data, device):
```

Comment:
docstring.

Reply:
Docstring is in L61 (GSgnnLinkPredictionModelInterface)